components/style/properties/gecko.mako.rs

```diff
@@ -3301,9 +3301,9 @@ fn static_assert() {
 
     ${impl_animation_timing_function()}
 
-    <% scroll_snap_type_keyword = Keyword("scroll-snap-type", "none mandatory proximity") %>
-
+    <% scroll_snap_type_keyword = Keyword("scroll-snap-type", "None Mandatory Proximity") %>
```

Comment:
Just curious - what's with the title cases?

Reply:
This generates a `Match` statement. All this is a mess, I want to rip it off... :)

Reply:
I didn't remember it auto-capitalized the names, huh :)

Reply:
That is, before this, the enum variants were actually `T::none`, `T::mandatory`, `T::proximity`, now they have rusty case.

Reply:
Yep, got it 👍